src/main/resources/messages/RsBundle.properties

```diff
@@ -83,3 +83,9 @@ intention.Rust.ToggleFeatureIntention.family.name=Toggle feature state
 
 # copy of `untrusted.project.notification.execution.error` from `ExternalSystemBundle.properties`
 untrusted.project.notification.execution.error=Cargo project is untrusted, so its task cannot be executed.
+
+dialog.create.project.custom.add.template.title=Add Custom Template
+dialog.create.project.custom.add.template.url=Template URL
+dialog.create.project.custom.add.template.url.description=cargo-generate supported template. <a href="https://github.com/cargo-generate/cargo-generate/blob/master/TEMPLATES.md">Available templates</a>
+dialog.create.project.custom.add.template.name=Name
+dialog.create.project.custom.add.template.action.add=Add
```

Comment:
Let's try to keep properties in message bundle in sorted order, i.e. sort these new group itself (`Sort lines` action may be helpful here) and move the group after `group.*` messages.
Sorted order may be useful for manual search